data/newMigrations/018-ethnicities.js

```diff
@@ -0,0 +1,10 @@
+exports.up = (knex) => {
+    return knex.schema.createTable('ethnicities', function (tbl) {
+        tbl.increments('ethnicity_id').primary();
```

Comment:
`.primary()` is not needed, when using `.increments()` knex sets it as primary behind the scenes